src/components/InputField.jsx

```diff
@@ -0,0 +1,21 @@
+
+///////this code defines a flexible InputField component for form inputs
+//////Reusable form field component that handles both regular inputs and textareas with built-in error display.
+function InputField({ label, name, value, onChange, placeholder, type = 'text', isTextarea = false, error }) {
```

Comment:
Omdat een text-area wel echt aanzienlijk anders is dan een gewoon invoerveld (van type date, email, tekst, password etc.) maak je daar meestal een ander component van! Een textarea heeft bijvoorbeeld geen type attribuut en wel row en column attributen, wat gewone inputs niet hebben